packages/fetchai/skills/weather_client/dialogues.py

```diff
@@ -48,6 +50,15 @@ def __init__(self, dialogue_label: DialogueLabel, is_seller: bool) -> None:
         FipaDialogue.__init__(self, dialogue_label=dialogue_label, is_seller=is_seller)
         self.proposal = None  # type: Optional[Description]
 
+    @staticmethod
+    def role_from_first_message(message: Message) -> BaseDialogue.Role:
```

Comment:
think

Reply:
This method is needed for now. If I remove it the skill wouldn't work. It can be removed once a generic implementation for `role_from_first_message` is provided in the `custom_types` file for fipa. 

Note this implementation should work across all skills.